identity-apps-core/apps/authentication-portal/src/main/webapp/includes/init-url.jsp

```diff
@@ -167,28 +167,3 @@
         longwaitstatusURL = identityServerEndpointContextParam + "/longwaitstatus";
     }
 %>
-
-<%!
-    /**
-    * Validates a given URL string to ensure it's safe for use, particularly in contexts like redirects.
-    * It checks for several common security vulnerabilities and invalid states:
-    * <ul>
-    * <li>Ensures the URL is not the string "null" (case-insensitive).</li>
-    * <li>Ensures the URL is not null, empty, or consists only of whitespace.</li>
-    * <li>Prevents common trickery by disallowing URLs starting with "javascript:", "file:", "ftp:", or "data:"
-    * (case-insensitive).</li>
-    * </ul>
-    *
-    * @param url The URL string to be validated.
-    * @return {@code true} if the URL is considered safe and valid based on the defined criteria;
-    * {@code false} otherwise.
-    */
-    public boolean isAdvancedSanitizedURL(String url) {
```

Comment:
This method was added with this PR https://github.com/wso2/identity-apps/pull/8260, However as there is an existing BE validation for this we can remove this.